Makefile

```diff
@@ -12,7 +12,7 @@ bench:
 	dune exec -- ./bench/bench_stream.exe
 	dune exec -- ./bench/bench_semaphore.exe
 	dune exec -- ./bench/bench_cancel.exe
-	dune exec -- ./lib_eio_linux/tests/bench_noop.exe
+	if ocamlc -config | grep -q '^system: linux'; then dune exec -- ./lib_eio_linux/tests/bench_noop.exe; fi
```

Comment:
Probably not the best solution for Windows, but there are other non-Windows specific commands in other rules.